sdk/schemaregistry/Azure.Data.SchemaRegistry/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.4.0-beta.2 (Unreleased)
+## 1.4.0-beta.2 (2023-08-08)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added `SchemaRegistrySerializer` which serializes using JSON by default.
+- Added `SchemaRegistrySerializerOptions`.
```

Comment:
I don't think we need to detail the options separately from the type that uses them.